mycore-base/src/test/java/org/mycore/common/MCRJPATestCase.java

```diff
@@ -133,7 +133,7 @@ public void setUp() throws Exception {
     }
 
     public void exportSchema() throws IOException {
-        doSchemaOperation(schema -> "create schema " + schema);
+        doSchemaOperation(schema -> "create schema if not exists " + schema);
```

Comment:
this prevents some false flag warnings if one of the tests was not cleaned up properly due to another error.

Reply:
then it will leave test data in the database that is even harder to detect later

Reply:
it would be best, if we would start each test in a new environment, 
I've been looking some time into the database settings until I figured out this was not the problem